src/io/portaudio_io.ml

```diff
@@ -64,6 +63,8 @@ class output ~kind ~clock_safe ~start ~on_start ~on_stop ~infallible buflen
           ~name:"output.portaudio" ~output_kind:"output.portaudio" val_source
           start as super
 
+    method private channels = self#ctype.Frame.audio
```

Comment:
Note for the future: at some point we might want to factor out the patterns that are repeated for I/O operators.

Reply:
Yes...